packages/fuel-indexer-lib/src/graphql/schema_gen.rs

```diff
@@ -0,0 +1,149 @@
+use crate::{
```

Comment:
Is there any particular reason this is in `fuel-indexer-lib` and not `fuel-indexer-macros` ? All of the codgen is 95% in `fuel-indexer-macros` (might be 100% but I'd have to double check).